apps/hubble/src/addon/src/store/message.rs

```diff
@@ -73,6 +73,9 @@ pub enum RootPrefix {
 
     /* Used to index fname username proofs by fid */
     FNameUserNameProofByFid = 27,
+
+    /* Used to index tags by target  */
```

Comment:
These pre and post-fixes are used to generate keys in the RocksDB key-value pairs